frontend/src/container/NewDashboard/DashboardVariablesSelection/DashboardVariableSelection.tsx

```diff
@@ -21,8 +27,38 @@ function DashboardVariableSelection(): JSX.Element | null {
 	const [update, setUpdate] = useState<boolean>(false);
 	const [lastUpdatedVar, setLastUpdatedVar] = useState<string>('');
 
+	const [variablesTableData, setVariablesTableData] = useState<any>([]);
+
 	const { role } = useSelector<AppState, AppReducer>((state) => state.app);
 
+	useEffect(() => {
+		if (variables) {
+			const tableRowData = [];
+			// eslint-disable-next-line sonarjs/no-unused-collection
+			const variableOrderArr = [];
+
+			for (const [key, value] of Object.entries(variables)) {
+				const { order, id } = value;
+
+				tableRowData.push({
+					key,
+					name: key,
+					...variables[key],
+					id,
+				});
+
+				if (order) {
+					variableOrderArr.push(order);
+				}
+			}
+
+			tableRowData.sort((a, b) => a.order - b.order);
+			variableOrderArr.sort((a, b) => a - b);
```

Comment:
Do we need `variableOrderArr` as we are not using is further?

Reply:
@Rajat-Dabade : Addressed ✅